### DISCUSSION TITLE
PDF to Audiobook converter

### DISCUSSION THREAD

**Discussion by JohnZolton** [suno-ai/bark#306] [Show and tell]
Convert any pdf to an audiobook, 100% locally with bark. Just drag and drop a pdf into the gui. On my 1070 I was able to convert 1.5 pages to 4 minutets of audio in 6 minutes. I parsed the text in 25 word (~12 second) chunks to minimize hallucinations and maintain fidelity to the text.

You can check it out [here](https://github.com/JohnZolton/pdf-2-mp3-two).

any feedback is welcome!

**Comment by adriens** [comment]
May I suggest to compress output to mp3 for better sharing on platforms ?

- https://github.com/suno-ai/bark/pull/305
